Porto-Tabs/schema.json

```diff
@@ -20,8 +20,7 @@
           },
           "Icon": {
             "type": "string",
-            "title": "Icon (optional) - Example: fab fa-android",
-            "helper": "Find values at: <a href=\"https://fontawesome.com/icons?d=gallery\" target=\"_blank\">FontAwesome</a>"
```

Comment:
@alejoroman0605  I think we should keep the helper, right?

Reply:
@WillStrohl it's already solved